cgi-bin/DW/BusinessRules/InviteCodes/DWS.pm

```diff
@@ -41,27 +41,39 @@ search_class, and adj_invites).
 # key is also used for long cat name (invitecodes.userclass.*), and first
 # argument is always max users to return.
 my %user_classes = (
-    basic_paid      => { search => \&_search_paystatus,
-                         search_arg => [ typeid => 3 ] },
-    premium_paid    => { search => \&_search_paystatus,
-                         search_arg => [ typeid => 4 ] },
-    permanent_paid  => { search => \&_search_paystatus,
-                         search_arg => [ permanent => 1 ] },
+    basic_paid => {
+        search     => \&_search_paystatus,
+        search_arg => [ typeid => 3 ]
+    },
+    premium_paid => {
+        search     => \&_search_paystatus,
+        search_arg => [ typeid => 4 ]
+    },
+    permanent_paid => {
+        search     => \&_search_paystatus,
+        search_arg => [ permanent => 1 ]
+    },
+
     # Users active in the last 30 days
-    active30d       => { search => \&_search_ctrk,
-                         search_arg => 30 },
+    active30d => {
+        search     => \&_search_ctrk,
+        search_arg => 30
+    },
+
     # Users with no invites left
-    noinvleft       => { search => \&_search_noinvleft }, # No search arg
-    # Users with no invites left and 1 invitee paid/perm/active in last 30 days
-    noinvleft_apinv => { search => \&_search_noinvleft_apinvitee,
-                         search_arg => 30 },
+    noinvleft => { search => \&_search_noinvleft },    # No search arg
+          # Users with no invites left and 1 invitee paid/perm/active in last 30 days
```

Comment:
Strange how it indented this comment (which pertains to the line below), but didn't indent the last similar comment at line 53/63. Maybe a blank line would help it understand that the comment goes with the next line.